### PR TITLE
docs: :pencil2: fix images on pypi

### PR DESCRIPTION
**Pull request by odulcy-mindee**
Attempt to fix https://github.com/mindee/doctr/issues/783

@felixdittrich92 Unfortunately, your previous fix failed because it's a link to an asset in a GitHub Release, which behaves differently than `https://raw.githubusercontent.com/...`. GitHub Release seems to always return an answer with `content-type=application/octet-stream`, which fools PyPi. Let's try by putting the file directly in the repo.

~~Also, I re-added `pip install` at the top of the README.md, I think it makes sense to have it on top to show it's (now) easy to install an up-to-date version of docTR. This point is open to discussion, of course.~~

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -73,4 +73,4 @@ body:
 - type: markdown
   attributes:
     value: >
-      Thanks for helping us improve the library!
```
Comment (felixT2K) [inline .github/ISSUE_TEMPLATE/bug_report.yml:76]: please revert :)

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -30,4 +30,4 @@ body:
 - type: markdown
   attributes:
     value: >
-      Thanks for contributing 🎉
```
Comment (felixT2K) [inline .github/ISSUE_TEMPLATE/feature_request.yml:33]: same

File: README.md [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
 
+
```
Comment (felixT2K) [inline README.md:7]: don't think the pypi badge and installation guide inside the docs are enough ? :)

File: notebooks/README.rst [hunk 4]
```diff
@@ -14,4 +14,4 @@ Here are some notebooks compiled for users to better leverage the library capabi
 +--------------------------------------------------------------------------------------------------------+----------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
 | `[Artefact detection] <https://github.com/mindee/notebooks/blob/main/doctr/artefact_detection.ipynb>`_ | Object detection for artefacts in documents  | .. image:: https://colab.research.google.com/assets/colab-badge.svg                                                 |
 |                                                                                                        |                                              |        :target: https://colab.research.google.com/github/mindee/notebooks/blob/main/doctr/artefact_detection.ipynb  |
-+--------------------------------------------------------------------------------------------------------+----------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
```
Comment (felixT2K) [inline notebooks/README.rst:17]: same :)

**Review by felixT2K — CHANGES_REQUESTED** [review]
Hey @odulcy-mindee 👋🏼,

thanks for the fix.
I have always tried to avoid exactly this case because with each installation you will also download the images 😅 
But at the end i think without uploading the images to a file server we have no chance to fix this unfortunately :/ 
(In short i agree that the only way is to keep the images in a docs folder just a few style changes please 👍🏼 )

File: README.md (hunk 3 above)
Comment (felixT2K) [reply README.md:7]: @odulcy-mindee https://github.com/mindee/doctr#installation

**Comment by odulcy-mindee** [comment]
> I have always tried to avoid exactly this case because with each installation you will also download the images 😅

Using a git-clone, yes :/
But for PyPi, it's fine

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (odulcy-mindee) [reply .github/ISSUE_TEMPLATE/bug_report.yml:76]: Don't why, it's pre-commit which did that

**Comment by felixT2K** [comment]
> > I have always tried to avoid exactly this case because with each installation you will also download the images 😅
> 
> Using a git-clone, yes :/ But for PyPi, it's fine

Yes, sorry, it was probably a bit unclear 😅

File: README.md (hunk 3 above)
Comment (odulcy-mindee) [reply README.md:7]: @felixT2K We can add a link to this header instead, wdyt ?

Comment (felixT2K) [reply README.md:7]: That would be fine but than moving it as first point after the Quicktour headline ?

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (felixT2K) [reply .github/ISSUE_TEMPLATE/bug_report.yml:76]: I see my pre-commit was out of date (after autoupdate it does the same ^^)

File: README.md (hunk 3 above)
Comment (odulcy-mindee) [reply README.md:7]: @felixT2K Well, in the end, I reverted that. I'm eager to update the README.md but let's do that in another PR.

**Review by felixdittrich92 — APPROVED** [review]
Thanks ☺️

**Pull request merged**